### ISSUE TITLE
Ability to configure auto center feature

### ISSUE THREAD

**Issue by zapfdk** [katspaugh/wavesurfer.js#1699]
### Wavesurfer.js version(s):
2.1

### Browser and operating system version(s):
Firefox and Chrome, Fedora

### Code needed to reproduce the issue:
{autoCenter = true}

### Use behaviour needed to reproduce the issue:
With autoCenter enabled, when seeking to left side of waveform, cursor only centers very slowly.

Is there a way or an option to center the cursor immediately on playback? I tried it with calling the seekAndCenter function at the seek Event, but of course this only leads to an infinite calling of the seek event.

**Comment by zapfdk** [comment]
Ok, I guess I tracked down the problem. When using the autoCenter option, the cursor is centered at a fixed rate by the drawer object. This leads to a really slow centering process for long audio files. 
As there is an option for immediate centering which is used when seeking or zooming, as a workaround I rewrote the function call to always immediately center the cursor. 

Could we make this an additional parameter while creating the wavesurfer object?
So in addition to 
`autoCenter: true` 
we could have something like
`centeringRate: 123,
immediateAutoCenter: true
`
